### DISCUSSION TITLE
MimicChest ID

### DISCUSSION THREAD

**Discussion by Insanguu** [Jukkales/NecroLens#13] [Ideas]
Hey! Where does the object ID of MimicChest come from? I have seen one instance where the overlay was correct and showed the title on a random chest once (I believe it was bronze). However, in many other situations it was not shown. Is there some inconsistency with the IDs? I have no tools to confirm those object IDs. If so, wouldn't it make sense to confirm chests with "normal" bronze/silver/gold chest IDs as they might always be the same and then consider every different chest as a hidden MimicChest?

**Comment by Jukkales** [comment]
Only mimics on lower levels in bronze chests can be shown in the overlay.
It is designed be SE that bronze mimics are send to the client as real mimic chest. 
Silver and goldchests are always hidden and cannot be seen.

**Reply by Insanguu** [reply]
Thanks for the info! Would have been a nice addition otherwise
